incubator/java-openliberty/templates/kafka/README.md

```diff
@@ -0,0 +1,76 @@
+# Kafka template for Open Liberty
+
+This is a template for the development of Liberty applications that can connect to Kafka using MicroProfile Reactive Messaging. There is a simple StarterApplication that enables basic production and consumption of events. The instructions for getting this up and running are below. As part of this, we have included a `docker-compose.yaml` file which will start a local Kafka and Zookeeper. 
+
+
+## Getting Started with the StarterApplication.
+
+### 1. Create a new folder and initialize it using appsody init:
+
+
+```
+mkdir test-appsody-reactive
+cd test-appsody-reactive
+appsody init java-openliberty reactive
+```
+
+### 2. Start Kafka and Zookeeper
+
+In order to run the StarterApplication we will need to start a Kafka and Zookeeper. For this, you can use the provided `docker-compose.yaml`.
+
+
+Start docker compose with 
+
+```docker-compose -f docker-compose.yaml up```
+
+If you run `docker network list`, you should see a new network, which will have the name of your directory with `_default` appended at the end. In this case it would be `test-appsody-reactive_default`.
```

Comment:
... you should see a new network with the name of your directory and the word `_default` appended. For example, `test-appsody-reactive_default`.